apps/mobile/src/components/headers/animated-header/animated-header.hooks.tsx

```diff
@@ -42,3 +51,16 @@ export function useAnimatedHeader(triggerAnimationYValue = 68) {
     },
   };
 }
+
+export function normalize(sourceRange: [number, number], targetRange: [number, number]) {
```

Comment:
Does this get used anywhere? I can't see it in this PR. 

- If it's not used lets remove it
- If it is used how about:
    - moving it into `./utils.ts`
    - renaming `animated-header.hooks` as `use-animated-header.ts`

There is only one hook in this file so I think it should be renamed anyway